src/ExtendedView.js

```diff
@@ -1,30 +1,50 @@
+import React from 'react'
+import {fetchSingleMoviePoster} from './apiCalls'
 import './ExtendedView.css'
 
-function ExtendedView(props) {
+class ExtendedView extends React.Component {
```

Comment:
This makes sense to have as class component since it is holding state that only this component is manipulating and using.